themes/default/content/docs/guides/continuous-delivery/azure-devops.md

```diff
@@ -97,6 +112,20 @@ Ubuntu agent, and some on a Windows agent. `pulumi` can be installed on these ag
 For the YAML-driven DevOps pipeline, the repository must contain the `azure-pipelines.yml` in the root of the repo for Azure DevOps to use it automatically.
 The following are samples only. You may choose to structure your configuration any way you like.
 
+### Log Pulumi Output as PR Comments
+
+The Pulumi task supports adding PR comments containing the log output from the Pulumi command that was executed in your build pipeline.
+Your project's build service user will need additional permissions to perform that action. Follow these steps to grant the build service user the `Contribute to pull requests` permission:
+
+- Navigate to the **Project Settings** page and click on **Repositories** under the **Repos** heading.
+- Select the repository where you will be using this feature and then click on the **Security** tab.
+- Now under the **Users** section find the build service user. If you are using the default build service user,
+the naming convention is `<Project name> Build Service` where `<Project name>` is your project's name.
+- Change the value of `Contribute to pull requests` to `Allow`.
+
+**Note**: This feature is only supported for builds triggered by pull requests created in git repositories hosted by Azure DevOps.
+Repositories hosted by external VCS such as Bitbucket, GitHub, GitLab are not supported at this time.
```

Comment:
may be worth using the note shortcode for this

Reply:
Done and moved it to the top of the subsection